client/my-sites/plugins/search-box-header/style.scss

```diff
@@ -37,13 +37,15 @@
 					color: var( --studio-black );
 				}
 
-				&.is-open input.search-component__input[type='search'] {
-					font-size: $font-body-small;
+				@media ( min-width: 660px ) {
+					&.is-open input.search-component__input[type='search'] {
```

Comment:
So in the end this was a css specificity issue ?

Reply:
`search` component by default uses `16px`. Instead of adding an override, can we let the default font-size of the search component? It's more a design decision, but let's challenge these decisions before adding more code.

Reply:
@vinimotaa what do you think about it ?

Reply:
Vini: should be the same desktop and mobile, font-size for search bar (16px for both).

Reply:
I removed the font size override in search component